### PR TITLE
Fixed a few small syntax issues

### PR DESCRIPTION
**Pull request by squarebraket**
This PR contains a few syntax fixes, mainly missed ';'s at the end of statements in the gulp "tasks" scripts.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,6 @@
   },
   "bugs": "https://github.com/gsong/es6-jspm-gulp-starter/issues/new",
   "jspm": {
-    "directories": {},
```
Comment (squarebraket) [inline package.json]: The installation process deleted this line, I'm not sure why...

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,6 @@
 .DS_STORE
 .build
 .tmp
+.idea/
```
Comment (gsong) [inline .gitignore]: What is `.idea/` for?

File: tasks/utils.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ export default (options) => {
   let builder = new jspm.Builder();
 
   return builder.buildSFX(INFILE, OUTFILE, options);
-}
+};
```
Comment (gsong) [inline tasks/utils.js]: Function declarations are blocks and not statements, so shouldn't end with semicolon.

File: .gitignore (hunk 2 above)
Comment (squarebraket) [inline .gitignore]: .idea/ is the folder WebStorm creates to store IDE setting for each project

File: tasks/utils.js (hunk 3 above)
Comment (squarebraket) [inline tasks/utils.js]: Sorry, I'll remove this mistake.

File: .gitignore (hunk 2 above)
Comment (gsong) [inline .gitignore]: I think this should just go into your global gitignore. Can you please remove this then the rest of the PR is good to go.

File: package.json (hunk 1 above)
Comment (gsong) [inline package.json]: jspm itself is still evolving. It's probably just due to its own internal spec changing. That's why it's best to not change `package.json` and `config.js` by hand if possible.

Comment (squarebraket) [inline package.json]: Ok sounds good, I didn't change these files at all.

File: .gitignore (hunk 2 above)
Comment (squarebraket) [inline .gitignore]: done

**Pull request merged**